solidity/interfaces/adapters/IUniswapV3Adapter.sol

```diff
@@ -50,9 +50,9 @@ interface IUniswapV3Adapter is ITokenPriceOracle {
    * @notice Emitted when support is updated (added or modified) for a new pair
    * @param tokenA One of the pair's tokens
    * @param tokenB The other of the pair's tokens
-   * @param pools The pools assigned to the pair
+   * @param preparedPools The amount of pools that were prepared to support the pair
```

Comment:
Instead of building a custom array just to emit it in the event, I thought it was easier and cheaper to just emit the amount of prepared pools